engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -38,6 +38,7 @@
 import javax.naming.ConfigurationException;
 
 import com.cloud.agent.api.AttachOrDettachConfigDriveCommand;
+import com.cloud.dc.*;
```

Comment:
@borisroman Can I prevent IntelliJ changing this? Because when I change it back, it is automatically changed again :-)